watson-developer-cloud/discovery-components
# fix: better handling of tables in processDoc

#### What do these changes do/fix?
ProcessDoc is now able to add top level tables to the tables list.

https://zenhub.ibm.com/app/workspaces/discovery-unified-5728bf39d2c20f776e05d0ef/issues/watson-discovery/docviz-squad-issue-tracker/252

#### How do you test/verify these changes?
Run `yarn test:watch processDoc` and see that the newly added test passes. If you were to run the test on develop it would not.
